app/src/main/java/com/example/stromtracker/ui/geraete/GeraeteFragment.kt

```diff
@@ -189,7 +185,10 @@ class GeraeteFragment : Fragment(), View.OnClickListener {
                 buttonSortVerbrauch.paintFlags = Paint.UNDERLINE_TEXT_FLAG
                 buttonSortName.paintFlags = 0
                 buttonSortRaum.paintFlags = 0
-
+                //TODO schönere Lösung?
+                buttonSortRaum.typeface = Typeface.DEFAULT_BOLD
+                buttonSortName.typeface = Typeface.DEFAULT_BOLD
+                buttonSortVerbrauch.typeface = Typeface.DEFAULT_BOLD
```

Comment:
Typeface auf Bold setzt die Schrift der Buttons auf fett oder?
In XML gibt es dafür das Setting 'android:textStyle="bold" ' ich weiß aber nicht, ob du das in Kotlin so verwenden kannst.

Reply:
Wenn alle Buttons **immer** Fett geschrieben sein sollen, könntest du es auch ins XML schreiben, oder in die Funktion (siehe Comment on lines 199 to 204)?

Reply:
Bin mir nicht sicher ob das durch das Unterstreichen das nicht zurückgesetzt wird, wenn ich das z.b direkt am Anfang setze (wenn ich die Buttons initalisiere), dann wird das nicht übernommen sobald sortiert wurde.
Habe aber auch ein Kommentar gemacht, ob das eventuell schöner geht, aber das kann man danach noch testen, vielleicht kriegst du es ja schöner hin.